### PR TITLE
fix index out of bounds

### PR DESCRIPTION
**Pull request by exokortex**
fix index out of bounds in case capstone returns unexpected disassembly result

### CODE REVIEW

File: libr/anal/p/anal_arm_cs.c [hunk 1]
```diff
@@ -2107,6 +2107,8 @@ r4,r5,r6,3,sp,[*],12,sp,+=
 		break;
 	case ARM_INS_BFI:
 	{
+		if (OPCOUNT() < 3 || ! ISIMM (3))
```
Comment (radare) [inline libr/anal/p/anal_arm_cs.c]: add braces

File: libr/anal/p/anal_arm_cs.c [hunk 2]
```diff
@@ -2117,6 +2119,8 @@ r4,r5,r6,3,sp,[*],12,sp,+=
 	}
 	case ARM_INS_BFC:
 	{
+		if (OPCOUNT() < 2 || ! ISIMM (2))
```
Comment (radare) [inline libr/anal/p/anal_arm_cs.c]: add braces

File: libr/anal/p/anal_arm_cs.c (hunk 1 above)
Comment (exokortex) [reply libr/anal/p/anal_arm_cs.c]: done

File: libr/anal/p/anal_arm_cs.c (hunk 2 above)
Comment (exokortex) [reply libr/anal/p/anal_arm_cs.c]: done

**Pull request merged**